app/views/layouts/application.html.erb

```diff
@@ -7,12 +7,21 @@
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= favicon_link_tag asset_path('logo.png') %>
```

Comment:
Great job setting this up 👍 It's really good that you've gone the extra mile there with favicon and icons. It's a good instinct to have when coding and will serve you well.